Programowanie Obiektowe/1.1Operatory/zad3/Program.cs

```diff
@@ -0,0 +1,10 @@
+﻿int a = 0;
+int b = 1;
+int c = 2;
+
+var tautology1 = a == b || a != b ;
+Console.WriteLine(tautology1);
+var tautology2 = !(a > b && a <= b);
+Console.WriteLine(tautology2);
+var tautology3 = !false;
+Console.WriteLine((a != c && b != c) == (a != c || b != c));
```

Comment:
Nie wiem czy tak to powinno wyglądać? Zobacz i ewentualnie popraw.

Reply:
[6c979a3](https://github.com/broniuu/Praktyki2022/pull/2/commits/6c979a303266dab1c48fe12bac5690d4b1dcb2ec)